include/psa/crypto_extra.h

```diff
@@ -2069,6 +2069,264 @@ static inline struct psa_pake_operation_s psa_pake_operation_init(void)
     return v;
 }
 
+typedef enum {
```

Comment:
We have avoided using enumerations in the public Crypto API, and prefer to use a set of macro definitions. This makes implementation-specific extensions easier, and avoids some complications with the data size of an enumeration.

Please make `psa_key_data_format_t` a integer typedef, and the values a set of macros.